plugin/src/main/java/org/opensearch/ml/breaker/DiskCircuitBreaker.java

```diff
@@ -32,17 +32,17 @@ public DiskCircuitBreaker(long threshold, String diskDir) {
 
     @Override
     public String getName() {
-        return  ML_DISK_CB;
+        return ML_DISK_CB;
     }
 
     @Override
     public boolean isOpen() {
         try {
             return AccessController.doPrivileged((PrivilegedExceptionAction<Boolean>) () -> {
-                return (new File(diskDir).getFreeSpace()/1024/1024/1024) < getThreshold();  // in GB
+                return (new File(diskDir).getFreeSpace() / 1024 / 1024 / 1024) < getThreshold();  // in GB
```

Comment:
minor: we can create constant for the disk threshold rather than recalculate for every request.
```
public static final long DEFAULT_FREE_DISK_SPACE_THRESHOLD = 5l * 1024 * 1024 * 1024;
```

Reply:
It's from original code not for this PR. Sure, I can add it.

Reply:
I prefer to add a contant only for 1024 * 1024 * 1024, as the original logic is using G as default unit.